charts/argo-cd/templates/argocd-repo-server/deployment.yaml

```diff
@@ -194,8 +183,18 @@ spec:
         name: var-files
       - emptyDir: {}
         name: plugins
-      {{- if .Values.repoServer.initContainers }}
       initContainers:
+      - command:
+        - cp
+        - -n
+        - /usr/local/bin/argocd
+        - /var/run/argocd/argocd-cmp-server
+        image: {{ default .Values.global.image.repository .Values.repoServer.image.repository }}:{{ default (include "argo-cd.defaultTag" .) .Values.repoServer.image.tag }}
+        name: copyutil
```

Comment:
Well spotted. Shall we add this to the conditional as well? :thinking:

Reply:
I don't think so. It is unrelated to the `initContainers` section from the values file.  

For some users, a new variable to disable this init container could be useful.

Reply:
Should I add this option? Otherwise, what is the plan of going forward with this PR? For me, it's a blocking issue.

Reply:
I reckon if this is not needed by all users let's add a variable to enable it. I'd suggest we make a default aligned with what upstream Kustomize is deploying.

Reply:
Ok. I added `enableCopyutilInitContainer` to solve this.